tools/Makefile.am

```diff
@@ -3,6 +3,9 @@ EXTRA_DIST =
 CLEANFILES =
 DISTCLEANFILES =
 
+libeosshellcontentdir = $(pkgpythondir)
+libeosshellcontent_PYTHON = libeosshellcontent.py
```

Comment:
I think you missed my comments here before. Hmm, actually I don't see them so maybe I forgot to press a button or something.

* Putting this in `pkgpythondir` will be broken since that would make it `site-packages/eos-shell-content/libeosshellcontent.py`, which is invalid since `eos-shell-content` has `-`s in it. Just using `python_PYTHON =` will put it directly in `site-packages`, I believe.

* I don't think the `lib` prefix in `libeosshellcontent.py` is necessary. This is almost never done with python modules (look at all the other stuff you're importing). I think just `eosshellcontent.py` would be nicer.